test/cbmc/proofs/initFileTransfer_Mqtt/initFileTransfer_Mqtt_harness.c

```diff
@@ -0,0 +1,64 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/**
+ * @file initFileTransfer_Mqtt_harness.c
+ * @brief Implements the proof harness for initFileTransfer_Mqtt function.
+ */
+/* Include headers for mqtt interface. */
+#include "ota_mqtt_private.h"
+
+/* Maximum size of the pBuffer for initFileTransfer_Mqtt function. */
+#define TOPIC_STREAM_DATA_BUFFER_SIZE    96U
+
+
+/* Stub required to combine a set of strings(to form a topic). */
+size_t __CPROVER_file_local_ota_mqtt_c_stringBuilder( char * pBuffer,
+                                                      size_t bufferSizeBytes,
+                                                      const char * strings[] )
+{
+    size_t stringLength;
+
+    __CPROVER_assert( pBuffer != NULL,
+                      "Unable to use pBuffer: passed pointer value is NULL." );
+
+    __CPROVER_assert( strings != NULL,
+                      "Unable to use strings: passed pointer value is NULL." );
+
+    /* The size of the static pbuffer is declared inside initFileTransfer_Mqtt
+     * function. */
+    __CPROVER_assume( stringLength > 0U && stringLength < TOPIC_STREAM_DATA_BUFFER_SIZE );
+
+    return stringLength;
+}
+
+/* Stub to a user defined MQTT-subscribe function. */
+OtaMqttStatus_t subscribe( const char * pTopicFilter,
+                           uint16_t topicFilterLength,
+                           uint8_t ucQoS )
+{
+    OtaMqttStatus_t status;
+
+    return status;
+}
+
+/*********************************************************/
+
+void initFileTransfer_Mqtt_harness()
+{
+    OtaAgentContext_t agent;
+    OtaFileContext_t filecontext;
+    OtaInterfaces_t pOtaInterface;
```

Comment:
This variable isn't a pointer, can you remove p from the start of it?

Reply:
Updated in f99d77c